flutter-gui-tests/build.gradle

```diff
@@ -10,14 +10,17 @@ repositories {
   mavenLocal()
   mavenCentral()
   jcenter()
+  // unpack flutter-intellij.zip and rename the jar file to io.flutter.jar in this directory
+  flatDir dirs: ['../releases/release_master/2019.1']
 }
 
 dependencies {
   compile "org.jetbrains.kotlin:kotlin-reflect"
   testCompile group: 'junit', name: 'junit', version: '4.12'
+  testCompile fileTree(include: ['*.jar'], dir: '../releases/release_master/2019.1')
```

Comment:
can we make 2019.1 a variable so we don't have to update it here when intellij gets a new release?